modules/icon/react/spec/Icon.spec.tsx

```diff
@@ -8,18 +8,8 @@ import {matchers} from 'jest-emotion';
 expect.extend(matchers);
 
 describe('Icon component', () => {
-  test('Sets size correctly', () => {
```

Comment:
It makes more sense to me to test CSS style rules in a snapshot test (and it's way easier).